service/arm_templates/security_groups_setup.json

```diff
@@ -67,27 +70,13 @@
               "protocol": "*",
               "sourcePortRange": "*",
               "destinationPortRange": "*",
-              "sourceAddressPrefix": "[parameters('masterSubnetCidr')]",
+              "sourceAddressPrefix": "*",
```

Comment:
Simplify here. Allow all outbound traffic.

Alternative would be allow VMs and Pods subnets and deny anything else. But i don't really care about outbound.